### ISSUE TITLE
Modal and autocomplete form

### ISSUE THREAD

**Issue by benjamin-bergia** [picturepan2/spectre#315]
Hi, first thanks for this great framework.
I have been spending quite some time, trying to get an autocomplete form to behave properly in a modal.
Pretty sure that I am doing something wrong, but I cannot see anything abnormal
Here is a quick [example](https://jsfiddle.net/k0evcv9r/).

What would be the proper way to embed the autocomplete form in my modal without having this scrollbar?

**Comment by picturepan2** [comment]
@benjamin-bergia I will add this.

**Comment by igeligel** [comment]
The problem is the absolute positioning of the menu in the autocomplete form ([source](https://github.com/picturepan2/spectre/blob/c811cb46631e66772a108c393f6ef53a83ca010b/src/_autocomplete.scss#L32)).

I could fix it by overriding the style but i think it is not a proper solution. Any reason you have used absolute positioning for this?

Anyway [here is the solution](https://jsfiddle.net/soocu7g6/) for the example.

**Comment by picturepan2** [comment]
@igeligel Yes, fixing it is easy, but making it a common pattern is difficult. Still thinking.
